### PR TITLE
change alpha1 in the Serverless tutorials

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- change alpha1 to alpha2 in Create an inline Function
- change alpha1 to alpha2 in Create a Git Function
- change alpha1 to alpha2 in Override runtime image
- change alpha1 to alpha2 in Inject envs

**Related issue(s)**
See also #16375

### CODE REVIEW

File: docs/03-tutorials/00-serverless/svls-14-inject-envs.md [hunk 1]
```diff
@@ -103,46 +103,66 @@ kubectl create secret generic  my-secret  --from-literal secret-env="I come from
...
+             memory: 64Mi
+     runtime: nodejs16
+     source:
+       inline:
+         dependencies: |-
+           {
+            "name": "example-1",
+            "version": "0.0.1",
+            "dependencies": {
+              "request": "^2.85.0"
```
Comment (anoipm) [inline docs/03-tutorials/00-serverless/svls-14-inject-envs.md:145]: IMO it's something wrong with dependencies - further analysis needed

Comment (anoipm) [inline docs/03-tutorials/00-serverless/svls-14-inject-envs.md:145]:
```suggestion
```
We don't need dependencies

File: docs/03-tutorials/00-serverless/svls-14-inject-envs.md [hunk 2]
```diff
@@ -103,46 +103,66 @@ kubectl create secret generic  my-secret  --from-literal secret-env="I come from
...
             }
```
Comment (anoipm) [inline docs/03-tutorials/00-serverless/svls-14-inject-envs.md:146]:
```suggestion
```
We don't need dependencies

File: docs/03-tutorials/00-serverless/svls-14-inject-envs.md [hunk 3]
```diff
@@ -103,46 +103,66 @@ kubectl create secret generic  my-secret  --from-literal secret-env="I come from
...
-    EOF
-    ```
+           }
```
Comment (anoipm) [inline docs/03-tutorials/00-serverless/svls-14-inject-envs.md:147]:
```suggestion
```
We don't need dependencies

File: docs/03-tutorials/00-serverless/svls-14-inject-envs.md [hunk 4]
```diff
@@ -103,46 +103,66 @@ kubectl create secret generic  my-secret  --from-literal secret-env="I come from
...
+         source: |-
+           module.exports = {
+               main: function (event, context) {
+                   envs = ["env1", "env2", "env3"]
+                   envs.forEach(function(key){
+                       console.log()
```
Comment (anoipm) [inline docs/03-tutorials/00-serverless/svls-14-inject-envs.md:153]:
```suggestion
                       console.log(`${key}:${readEnv(key)}`)
```
Previous version printed env values in this place

File: docs/03-tutorials/00-serverless/svls-14-inject-envs.md [hunk 5]
```diff
@@ -103,46 +103,47 @@ kubectl create secret generic  my-secret  --from-literal secret-env="I come from
...
+             name: my-secret
+     runtime: nodejs16
+     source:
+       inline:
+         source: |-
+           module.exports = {
+               main: function (event, context) {
+                   envs = ["env1", "env2", "env3"]
+                   envs.forEach(function(key){
+                       console.log(`${key}:${readEnv(key)}`)
```
Comment (anoipm) [inline docs/03-tutorials/00-serverless/svls-14-inject-envs.md:134]:
```suggestion
                       console.log(\`${key}:${readEnv(key)}\`)
```
Shell try to evaluate expression inside backticks - we should escape it

**Review by anoipm — APPROVED** [review]

File: docs/03-tutorials/00-serverless/svls-02-create-git-function.md [hunk 6]
```diff
@@ -62,19 +62,24 @@ Follow these steps:
...
-    apiVersion: serverless.kyma-project.io/v1alpha1
-    kind: GitRepository
-    metadata:
-      name: $GIT_FUNCTION
-      namespace: $NAMESPACE
-    spec:
-      url: "https://github.com/kyma-project/examples.git"
-    EOF
-    ```
+3. Create a a Function CR that specifies the Function's logic and points to the directory with code and dependencies in the given repository. It also specifies the Git repository metadata:
```
Comment (IwonaLanger) [inline docs/03-tutorials/00-serverless/svls-02-create-git-function.md:65]:
```suggestion
3. Create a Function CR that specifies the Function's logic and points to the directory with code and dependencies in the given repository. It also specifies the Git repository metadata:
```

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**